tests/writer.rs

```diff
@@ -87,7 +87,7 @@ mod tests {
     }
 
     #[test]
```

Comment:
Would it be worth making another test for a non-contiguous array, that is in neither C nor F order?

Reply:
Excellent idea. I'll add that to my list and contribute it in another PR.